frontend/public/components/overview/resource-overview-page.tsx

```diff
@@ -9,19 +9,38 @@ import { NetworkingOverview } from './networking-overview';
 import { PodsOverview } from './pods-overview';
 import { resourceOverviewPages } from './resource-overview-pages';
 import { OverviewItem } from '@console/shared';
+import * as plugins from '../../plugins';
 
 const { common } = Kebab.factory;
 
-export const OverviewDetailsResourcesTab: React.SFC<OverviewDetailsResourcesTabProps> = ({
-  item: { buildConfigs, routes, services, pods, obj },
-}) => (
-  <div className="overview__sidebar-pane-body">
-    <PodsOverview pods={pods} obj={obj} />
-    <BuildOverview buildConfigs={buildConfigs} />
-    <NetworkingOverview services={services} routes={routes} />
-  </div>
+const getResourceTabSectionComp = (t) => (props) => (
+  <AsyncComponent {...props} loader={t.properties.loader} />
 );
 
+const getPluginTabSectionResource = (item) => {
+  const sections = plugins.registry
+    .getOverviewTabSections()
+    .filter((section) => item[section.properties.key]);
+  return sections.map((section) => getResourceTabSectionComp(section));
```

Comment:
You could just chain it all together:
```
  return plugins.registry
    .getOverviewTabSections()
    .filter((section) => item[section.properties.key])
    .map((section) => getResourceTabSectionComp(section));
```

Reply:
fixed in https://github.com/openshift/console/pull/3056/commits/8e27cbfc8c376c202bd2471c788bf0401b8dcb9d

Reply:
fixed in https://github.com/openshift/console/pull/3056/commits/8e27cbfc8c376c202bd2471c788bf0401b8dcb9d